.github/workflows/build.yml

```diff
@@ -21,22 +21,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Install Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-          components: rustfmt
-
       - name: Fail on warnings
         run: sed -i 's,//deny_warnings_placeholder_for_ci,#![deny(warnings)],' src/lib.rs macros/src/lib.rs
 
+      - name: Add Rust component
+        run: rustup component add rustfmt
```

Comment:
is this even needed? i think this component is present by default (at least i'm using `cargo fmt` in my GH CI builds w/o installing it first). previously you used `profile: minimal` which might've led to rustfmt not being present, but the default image has the full rust toolchain installed.

Reply:
I've set the profile to minimal on the other jobs, but "missed" it here, maybe that's another way of doing it, just not doing minimal and then not having to add the component instead.

If profile is `default`, adding an already existing component is a no-op? Will check the log

Reply:
Looks like a no-op, just states it's up to date and takes 0 seconds :D

Reply:
```suggestion
```

Reply:
Got the opportunity to try pushing the "suggestions" to a new level, noted they now support batching suggestions together :)

Thank you for pointing out the redundant stuff :+1: